tools/ipa/Makefile

```diff
@@ -0,0 +1,34 @@
+# A Self-Documenting Makefile: http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+
+SPECTRAL_VERSION=${SPECTRAL_VERSION:-6.11.1}
+SPECTRAL_CORE_VERSION=${SPECTRAL_CORE_VERSION:-1.19.1} # If we need to use spectral core .js
+
+.PHONY: deps
+deps:  ## Download go module dependencies
+	@echo "==> Installing npx..."
+	npm install -g npx
```

Comment:
if you gonan use npm can you move to a package json instead of a make file, also postman has a package json, let's please have a single package json and not multiple ones

Reply:
Yes, was copying the existing spectral makefile for now. Do you mean having the same package json for postman + ipa tool?

Reply:
>Do you mean having the same package json for postman + ipa tool?

Yes a single source for dependencies in the repo, ie a mono repo approach